src/domains/project/service/project.service.impl.ts

```diff
@@ -86,7 +86,25 @@ export class ProjectServiceImpl implements ProjectService {
       return acum;
     }, filteredAssignees);
 
-    return { stages: filteredStages, assignees: filteredAssignees, priorities: filteredPriorities };
+    return {
+      stages: filteredStages
+        .sort((a, b) => {
+          if (a.type !== b.type) {
+            return StageTypeOrder[a.type] - StageTypeOrder[b.type];
+          }
+          return a.position - b.position;
+        })
+        .map((stage) => new StageDTO(stage)),
```

Comment:
Why don't you return `StageExtendedDTO` instead? Why removing one single attribute?

Reply:
Frontend team asked to remove this attribute to avoid possible missunderstangins.